### PR TITLE
[TRANSLATION] Hebrew Translations For Setup

### PR DESCRIPTION
**Pull request by shytzedaka**
I have Fixed some of the Translations and Added some new ones.

### CODE REVIEW

**Comment by peterooch** [comment]
> I fixed the Order in the sentence when English and Hebrew are Mixed (Symbols and English words revert to the start of the sentence)
It is in that order because the window is LTR (Left-to-right) rendered, if the window was RTL (Right-to-Left) your changes will look "incorrect". BiDi algorithm will draw the text correctly (for us Hebrew speakers) if the window is RTL.
Side note: RTL mirroring is broken and it would be preferred to keep to LTR for now.

File: base/setup/reactos/lang/he-IL.rc [hunk 1]
```diff
@@ -1,65 +1,65 @@
...
-CAPTION "התקנת ReactOS"
+CAPTION "ReactOS התקנת"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "ברוכים הבאים לאשף ההתקנה של ReactOS", IDC_STARTTITLE, 115, 8, 195, 24
-    LTEXT "עדיין אי אפשר להתקין את ReactOS ישירות מהתקליטור הזה! נא אתחל מחדש מחשבך מהדיסק כדי להתקין את ReactOS.", IDC_STATIC, 115, 40, 195, 100
-    LTEXT "לחץ על סיום כדי לסיים את ההתקנה.", IDC_STATIC, 115, 169, 195, 17
+    LTEXT "ReactOS ברוכים הבאים לאשף ההתקנה של", IDC_STARTTITLE, 115, 8, 195, 24
+    LTEXT "ReactOS מהדיסק הזה ברגע זה!נע הפעל מחדש את המחשב מהדיסק הזה בשביל להתקין את ReactOS אתה לא יכול להתקין את", IDC_STATIC, 115, 40, 195, 100
+    LTEXT ".לחץ על 'סיום' בשביל לסיים את ההתקנה", IDC_STATIC, 115, 169, 195, 17
```
Comment (peterooch) [inline base/setup/reactos/lang/he-IL.rc]: Use double quotes "" instead of '

Comment (shytzedaka) [reply base/setup/reactos/lang/he-IL.rc]: Ah. will be fixing this commit soon.

**Comment by shytzedaka** [comment]
> Please revert the order changes because they are "visual order" instead of "logical order".

Please Check the New Commit and Changes.

**Comment by HBelusca** [comment]
Can't we use this trick https://git.reactos.org/?p=reactos.git;a=blob;f=base/setup/welcome/lang/he-IL.rc;hb=b8383b4546d230f854978c7642f124fc119e80fb#l6 to get the layout OK? (in case we don't want to depend on the `SetProcessDefaultLayout(LAYOUT_RTL);` API call)

**Comment by peterooch** [comment]
> Can't we use this trick https://git.reactos.org/?p=reactos.git;a=blob;f=base/setup/welcome/lang/he-IL.rc;hb=b8383b4546d230f854978c7642f124fc119e80fb#l6 to get the layout OK? (in case we don't want to depend on the `SetProcessDefaultLayout(LAYOUT_RTL);` API call)

Each dialog needs `EXSTYLE WS_EX_LAYOUTRTL`  to fix layout, and if mirroring was OK the they would have that...

**Pull request merged**